sql/updates/world/3.3.5/9999_99_99_world.sql

```diff
@@ -0,0 +1,47 @@
+
+-- Trash Mobs Creature Text
+DELETE FROM `creature_text` WHERE `CreatureID` IN (17899,17905,17919,17934,17932,17933,17936) AND `GroupID` IN (0,1);
+DELETE FROM `creature_text` WHERE `CreatureID`=17772 AND `GroupID`=7;
+INSERT INTO `creature_text`(`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(17899, 0, 0, '', 12, 0, 100, 0, 0, 0, 18450, 0, 'Shadowy Necromancer say slay'),
+(17899, 1, 0, '', 12, 0, 100, 0, 0, 0, 18448, 0, 'Shadowy Necromancer say death'),
+(17899, 1, 1, '', 12, 0, 100, 0, 0, 0, 18447, 0, 'Shadowy Necromancer say death'),
+(17899, 1, 2, '', 12, 0, 100, 0, 0, 0, 18446, 0, 'Shadowy Necromancer say death'),
+(17905, 0, 0, '', 12, 0, 100, 0, 0, 0, 18451, 0, 'Banshee say slay'),
+(17905, 0, 1, '', 12, 0, 100, 0, 0, 0, 18454, 0, 'Banshee say slay'),
+(17905, 1, 0, '', 12, 0, 100, 0, 0, 0, 18452, 0, 'Banshee say death'),
+(17905, 1, 1, '', 12, 0, 100, 0, 0, 0, 18453, 0, 'Banshee say death'),
+(17919, 0, 0, '', 12, 0, 100, 0, 0, 0, 18457, 0, 'Alliance Footman say slay'),
+(17919, 0, 1, '', 12, 0, 100, 0, 0, 0, 18456, 0, 'Alliance Footman say slay'),
+(17919, 1, 0, '', 12, 0, 100, 0, 0, 0, 18458, 0, 'Alliance Footman say death'),
+(17934, 0, 0, '', 12, 0, 100, 0, 0, 0, 18501, 0, 'Horde Headhunter say slay'),
+(17934, 0, 1, '', 12, 0, 100, 0, 0, 0, 18499, 0, 'Horde Headhunter say slay'),
+(17934, 0, 2, '', 12, 0, 100, 0, 0, 0, 18500, 0, 'Horde Headhunter say slay'),
+(17934, 1, 0, '', 12, 0, 100, 0, 0, 0, 18502, 0, 'Horde Headhunter say death'),
+(17932, 0, 0, '', 12, 0, 100, 0, 0, 0, 18486, 0, 'Horde Grunt say slay'),
+(17932, 0, 1, '', 12, 0, 100, 0, 0, 0, 18485, 0, 'Horde Grunt say slay'),
+(17932, 0, 2, '', 12, 0, 100, 0, 0, 0, 18488, 0, 'Horde Grunt say slay'),
+(17932, 0, 3, '', 12, 0, 100, 0, 0, 0, 18490, 0, 'Horde Grunt say slay'),
+(17932, 1, 0, '', 12, 0, 100, 0, 0, 0, 18487, 0, 'Horde Grunt say death'),
+(17933, 0, 0, '', 12, 0, 100, 0, 0, 0, 18493, 0, 'Tauren Warrior say slay'),
+(17933, 0, 1, '', 12, 0, 100, 0, 0, 0, 18494, 0, 'Tauren Warrior say slay'),
+(17933, 1, 0, '', 12, 0, 100, 0, 0, 0, 18495, 0, 'Tauren Warrior say death'),
+(17936, 0, 1, '', 12, 0, 100, 0, 0, 0, 18510, 0, 'Horde Shaman say slay'),
+(17936, 1, 0, '', 12, 0, 100, 0, 0, 0, 18496, 0, 'Horde Shaman say death'),
+(17772, 7, 0, '', 16, 0, 100, 0, 0, 0, 15310, 0, 'jaina hyjal horde base SUCCESS');
```

Comment:
Text should be filled and shouldn't create mismatches with broadcast_text, actually there should be an error for that like in npc_text (validation reasons)